worlds/stardew_valley/data/items.csv

```diff
@@ -941,15 +943,20 @@ id,name,classification,groups,mod_name
 10702,Resource Pack: 3 Travel Core,filler,RESOURCE_PACK,Magic
 10703,Preservation Chamber,filler,"RESOURCE_PACK,RESOURCE_PACK_USEFUL",Archaeology
 10704,Hardwood Preservation Chamber,filler,"RESOURCE_PACK,RESOURCE_PACK_USEFUL",Archaeology
-10705,Resource Pack: 3 Water Shifter,filler,RESOURCE_PACK,Archaeology
+10705,Resource Pack: 3 Water Sifter,filler,RESOURCE_PACK,Archaeology
 10706,Resource Pack: 5 Hardwood Display,filler,RESOURCE_PACK,Archaeology
 10707,Resource Pack: 5 Wooden Display,filler,RESOURCE_PACK,Archaeology
 10708,Grinder,filler,"RESOURCE_PACK,RESOURCE_PACK_USEFUL",Archaeology
 10709,Ancient Battery Production Station,filler,"RESOURCE_PACK,RESOURCE_PACK_USEFUL",Archaeology
-10710,Hero Elixir,filler,RESOURCE_PACK,Starde Valley Expanded
+10710,Hero Elixir,filler,RESOURCE_PACK,Stardew Valley Expanded

```

Comment:
Surprised that this existed with the typo for so long. We should make a test for that. I'll see what I can do